var/spack/repos/builtin/packages/delphes/package.py

```diff
@@ -18,6 +18,7 @@ class Delphes(CMakePackage):
     maintainers = ['drbenmorgan', 'vvolkl', 'selvaggi']
 
     version('master', branch='master')
+    version('3.4.3pre04', tag='3.4.3pre04')
     version('3.4.2', sha256='d46a7c5474de650befdb89377115feee31f1743107ceb3d8da699be9d48c097b')
```

Comment:
```suggestion
    version('3.4.2', sha256='d46a7c5474de650befdb89377115feee31f1743107ceb3d8da699be9d48c097b', preferred=True)
```
We don't want pre-release versions to be installed by default.